### PR TITLE
feat(woolwars): add woolwars stats

### PR DESCRIPTION
**Pull request by eatmyvenom**

### CODE REVIEW

**Comment by eatmyvenom** [comment]
I need to look further into the leveling system. The code there is similar the the bedwars one and so far does yield correct results but I would like to test further and have at least one prestige.

File: packages/schemas/src/player/gamemodes/woolgames/class.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    this.assists = data?.assists ?? 0;
+
+    this.powerups = data?.powerups_gotten ?? 0;
+    this.blocksBroken = data?.blocks_broken ?? 0;
+    this.woolPlaced = data?.wool_placed ?? 0;
+  }
+}
+
+export class WoolWarsStats {
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/woolgames/class.ts:53]: Merge this into `WoolGames`

File: packages/schemas/src/player/gamemodes/woolgames/index.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { APIData } from '@statsify/util';
+import { Color, ColorCode } from '../../../color';
+import { Field } from '../../../metadata';
+import { Progression } from '../../../progression';
+import { WoolWarsStats } from './class';
+import { getExpReq, getFormattedLevel, getLevel } from './util';
+
+export class WoolGames {
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/woolgames/index.ts:8]: Rename this to `WoolWars`

File: packages/schemas/src/player/gamemodes/woolgames/class.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  public powerups: number;
+
+  @Field()
+  public blocksBroken: number;
+
+  @Field()
+  public woolPlaced: number;
+
+  public constructor(data: APIData) {
+    this.wins = data?.wins ?? 0;
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/woolgames/class.ts:37]: you should be able to remove a lot of these ``?? 0`. Serialization/Deserialization will take care of it.

File: packages/schemas/src/player/gamemodes/woolgames/class.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+export class WoolWarsClass {
+  @Field()
+  public wins: number;
+
+  @Field()
+  public gamesPlayed: number;
+
+  @Field()
+  public losses: number;
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/woolgames/class.ts:13]: add `wlr`

File: packages/schemas/src/player/stats.ts [hunk 5]
```diff
@@ -116,5 +120,6 @@ export class PlayerStats {
     this.vampirez = new VampireZ(data?.stats?.VampireZ ?? {});
     this.walls = new Walls(data?.stats?.Walls ?? {});
     this.warlords = new Warlords(data?.stats?.Battleground ?? {});
+    this.woolgames = new WoolGames(data?.stats?.WoolGames ?? {});
```
Comment (jacobk999) [inline packages/schemas/src/player/stats.ts:123]: Rename to `woolwars`

File: packages/schemas/src/player/gamemodes/parkour/index.ts [hunk 6]
```diff
@@ -78,5 +81,6 @@ export class Parkour {
     this.tourney = getTime('Tourney');
     this.uhc = getTime('uhc');
     this.warlords = getTime('Warlords');
+    this.woolgames = getTime('WoolGames');
```
Comment (jacobk999) [inline packages/schemas/src/player/gamemodes/parkour/index.ts:84]: Rename to `woolwars`

**Review by jacobk999 — CHANGES_REQUESTED** [review]

**Comment by jacobk999** [comment]
It would also be nice if you added wool wars to https://github.com/Statsify/statsify/blob/main/packages/schemas/src/game.ts

**Comment by ugcodrr** [comment]
add it to ACHIEVEMENTS https://github.com/Statsify/statsify/tree/main/packages/schemas/src/achievements

thanks, Mr. venomous

File: packages/schemas/src/player/gamemodes/woolgames/index.ts (hunk 2 above)
Comment (eatmyvenom) [reply packages/schemas/src/player/gamemodes/woolgames/index.ts:8]: no

File: packages/schemas/src/player/gamemodes/woolgames/class.ts (hunk 1 above)
Comment (eatmyvenom) [reply packages/schemas/src/player/gamemodes/woolgames/class.ts]: ok

File: packages/schemas/src/player/gamemodes/parkour/index.ts (hunk 6 above)
Comment (eatmyvenom) [reply packages/schemas/src/player/gamemodes/parkour/index.ts:84]: no

File: packages/schemas/src/player/gamemodes/woolgames/class.ts (hunk 3 above)
Comment (eatmyvenom) [reply packages/schemas/src/player/gamemodes/woolgames/class.ts:37]: this is untrue :(!

**Comment by jacobk999** [comment]
Add it to: 
* https://github.com/Statsify/statsify/blob/main/packages/schemas/src/guild/expbygame.ts
* https://github.com/Statsify/statsify/tree/main/packages/schemas/src/achievements/index.ts
* https://github.com/Statsify/statsify/blob/main/packages/schemas/src/game.ts
* https://github.com/Statsify/statsify/blob/main/packages/schemas/src/gamecounts/index.ts

File: packages/schemas/src/player/gamemodes/woolgames/index.ts (hunk 2 above)
Comment (jacobk999) [reply packages/schemas/src/player/gamemodes/woolgames/index.ts:8]: It doesn't make sense to have the kits in the class if it is going to be called `WoolGames`. If more games are added then I will refactor it back to be `WoolGames`. Until then call it `WoolWars`.

**Review by jacobk999 — APPROVED** [review]

**Pull request merged**